.github/workflows/publish-single.yml

```diff
@@ -17,13 +17,14 @@ jobs:
     permissions:
       id-token: write # This is required for requesting the OIDC
     runs-on: ubuntu-latest
+    if: github.repository_owner == 'maplibre'
     steps:
       - uses: actions/checkout@v4
-      # Setup Dart SDK with JWT token
-      - uses: dart-lang/setup-dart@v1
+      - uses: subosito/flutter-action@v2
+        with:
+          cache: true
       - name: Install dependencies
-        run: dart pub get
-      # Publishing...
-      # skip validation due to https://github.com/dart-lang/setup-dart/issues/68
+        run: flutter pub get
+        # --force skips the y/N confirmation
       - name: Publish to pub.dev
-        run: dart pub publish --force --skip-validation
```

Comment:
We need to revert these changes, release is stuck here.

Reply:
https://github.com/maplibre/flutter-maplibre-gl/pull/426